contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -82,6 +82,7 @@
 #include "multidb.h"
 #include "tsql_analyze.h"
 #include "table_variable_mvcc.h"
+#include "commands/event_trigger.h"
```

Comment:
Is this include needed ? Which object is dependant on it.

Reply:
Just checked, not explicitly needed, removing it.